extensions/amp-carousel/0.1/slides.js

```diff
@@ -49,7 +49,7 @@ export class AmpSlides extends BaseCarousel {
     this.slides_.forEach((slide, i) => {
       this.setAsOwner(slide);
       // Only the first element is initially visible.
-      slide.style.display = i > 0 ? 'none' : 'block';
```

Comment:
I just need some context here: `display: none` is much better from a performance pov. But if we do want to render them, then this change is good.